examples/official-site/sqlpage/migrations/33_carousel_component.sql

```diff
@@ -0,0 +1,150 @@
+INSERT INTO component (name, description, icon, introduced_in_version)
+VALUES (
+        'carousel',
+        'A carousel is used to display multiple pieces of visual content without taking up too much space.',
+        'carousel-horizontal',
+        '0.18.0'
+    );
+
+INSERT INTO parameter (
+        component,
+        name,
+        description,
+        type,
+        top_level,
+        optional
+    )
+VALUES (
+        'carousel',
+        'name',
+        'Name of the carousel.',
+        'TEXT',
+        TRUE,
+        FALSE
+    ),
+    (
+        'carousel',
+        'title',
+        'Title of the carousel.',
```

Comment:
The docs does not make it clear what the difference between name and title is.

Reply:
I changed with 'An unique string to identify the caroussel component in the HTML page.' and 'A name to display at the top of the carousel.'